homeassistant/components/camera/arlo.py

```diff
@@ -158,15 +161,48 @@ def disable_motion_detection(self):
         self._motion_status = False
         self.set_base_station_mode(ARLO_MODE_DISARMED)
 
+    @staticmethod
+    def clean_attr(attr):
+        """Return unknown if attribute is None (non-subscriptable)."""
+        return str(attr) if attr is not None else STATE_UNKNOWN
+
     def update(self):
         """Add an attribute-update task to the executor pool."""
-        self.attrs[ATTR_BATTERY_LEVEL] = self._camera.get_battery_level
-        self.attrs[ATTR_BRIGHTNESS] = self._camera.get_battery_level
-        self.attrs[ATTR_FLIPPED] = self._camera.get_flip_state,
-        self.attrs[ATTR_MIRRORED] = self._camera.get_mirror_state,
-        self.attrs[
-            ATTR_MOTION] = self._camera.get_motion_detection_sensitivity,
+        # pylint: disable=W0212
+        base_stations = self._camera._session.base_stations
+
+        if not base_stations:
+            return None
+
+        # pylint: disable=W0212
+        base_stations[0]._refresh_rate = SCAN_INTERVAL.total_seconds()
+
+        base_stations[0].update()
+        self._camera.update()
+
+        battery_level = self.clean_attr(self._camera.get_battery_level)
```

Comment:
Do you need to drop the `get_` from these based on [this commit](https://github.com/tchellomello/python-arlo/commit/c58d96932c57069d660f5cb9e4d84bc81e7a8ef3)?

Reply:
Sorry! I'm seeing some weird errors while trying to fix an arlo issue. Thanks!

Reply:
Yes drop the `get_` from that attribute :+1:  Good point @mikesplain